README_docker.md

```diff
@@ -1 +1,233 @@
+
+#Install “dplyr” package version 1.0.10 with no upgrade
+RUN Rscript -e 'remotes::install_version("dplyr",upgrade="never", version = "1.0.10")'
+
+#Install “data.table” package version 1.4.1 with no upgrade
+RUN Rscript -e 'remotes::install_version("data.table",upgrade="never", version = "1.14.2")'
+
+#Install “comhis” package 
+RUN Rscript -e 'remotes::install_github("comhis/comhis")'
+
+#create directory (build_zone)
+RUN mkdir /build_zone
+
+#Copy all files to filesystem of image at path /build_zone
+ADD . /build_zone
+
+#set the working directory (to build_zone)
+WORKDIR /build_zone
+
+#remove directory (-rf option - without being prompted)
+RUN rm -rf /build_zone
+
+#copy all files to filesystem of the container at path /home/fennica
+COPY . /home/fennica
+ 
+#set working directory to /home/fennica
+WORKDIR /home/fennica
+
+#Execute the the R command to install 
+RUN R -e 'remotes::install_local(upgrade="never")'
+
+#Set working directory to the for all subsequent Dockerfile instructions
+WORKDIR /home/fennica/inst/examples
+
+#Change recursively to the group ID 0(root group) and recursively grant group permissions to be the same as the user’s.
+RUN chgrp -R 0 /home/fennica && \
+    chmod -R g=u /home/fennica
+
+#Execute the R command to render the the bookdown file and move _book file to directory public
+RUN R -e 'bookdown::render_book("index.Rmd", "bookdown::gitbook")' && \
+    mv _book /public
+
+#Starts new build stage with NGINX unprivileged base image
+FROM nginxinc/nginx-unprivileged:latest
+
+#Copy public folder from build stage to nginx public folder
+COPY --from=builder /public /usr/share/nginx/html
+
+#informs Docker that the container listens on the specified network ports at runtime
+EXPOSE 8080
+
+#Start NgInx service
+CMD ["nginx", "-g", "daemon off;"]
+```
+
+3.Build the container image using the following commands:
+
+```	
+	$cd fennica
+```
+Build the container image:
+```
+	$ docker build -t <docker-image-name>
+```
+After building the docker image we can check if the image exists using the commands such as $docker ps     or  $ docker ps -a. 
+Then we can share to a docker registry such as Dockerhub using the following commands.
+```
+docker tag <image id> your-dockerhub-user-name/image-name[:tag]
+docker login
+docker push your-dockerhub-user-name/image-name[:tag]
+```
+## Deployment on Rahti
+
+
+After pushing to the Dockerhub we use the following Deployment,service and route configurations and run the following command on the terminal.
+```
+echo 'apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: fennica
+spec:
+  selector:
+    matchLabels:
+      app: fennica
+  template:
+    metadata:
+      labels:
+        app: fennica
+    spec:
+      containers:
+      - name: fennica
+        image: "docker.io/your-dockerhub-user-name/image-name[:tag]"
+        resources: {}
+        ports:
+        - containerPort: 8080
+          protocol: TCP
```

Comment:
Does this work with the placeholder: "docker.io/your-dockerhub-user-name/image-name[:tag]" ?